fastboot-tests/fixtures/treeshake/ember-cli-build.js

```diff
@@ -0,0 +1,11 @@
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, {
+    'ember-bootstrap': {
+      whitelist: ['bs-button-group']
```

Comment:
Yes, there is a `blacklist`, but the current implementation relies on having either a `whitelist` or a `blacklist`, not both. And adding `whitelist: []` would exclude all components form the build, so probably not a good default! 😝

I don't know if it would be possible to add those, but commented out? I don't think the is a way to do that with AST, is there? But I personally am ok with how it is now. Users can still add those options if they need them...